settings/default/main.lua

```diff
@@ -90,6 +90,7 @@ xi.settings.main =
     -- CHARACTER CONFIG
     INITIAL_LEVEL_CAP              = 50, -- The initial level cap for new players.  There seems to be a hardcap of 255.
     MAX_LEVEL                      = 99, -- Level max of the server, lowers the attainable cap by disabling Limit Break quests.
+    MAX_MASTER_LEVEL               = 50, -- Maximum master level allowed. Warning: If this is enabled, SUBJOB_RATIO is overridden. -1 for disabled, 50 (RETAIL CAP)
```

Comment:
It shouldn't really override per se. Master levels add to the subjob level cap, which non standard ratios would have that cap at a different number is all. 

Still, I would advise server against combinations that cause subjob to go higher than 99 or higher than mainjob, something bound to break in an unsupported way which I will without hesitation call user error!

Reply:
basically think of them as "bonus levels" rather than this main lv = this exact sub lv